### ISSUE TITLE
Accessibility: 

### ISSUE THREAD

**Issue by DBAKITA** [jupyterhub/outreachy#129]
### What page is this for?

https://ipython.org/documentation.html

### WAVE accessibility report

Wave accessibility Report Link is https://github.com/jupyterhub/outreachy/issues/38#issuecomment-1277649881

![wave evaluation of ipython documentation](https://user-images.githubusercontent.com/87822858/198734035-8106ce8d-645b-4a2c-b5e8-8af52310971b.JPG)

I evaluated https://ipython.org/documentation.html using WAVE web accessibility evaluation tool.

Below are the findings and the suggested changes to improve the page's accessibility.

1. Language Missing or Invalid (The attribute is missing or is empty, or a lang attribute value is not a valid language identifier.)

Suggestion: The document language should be identified using the attribute with a vald value (e.g., ). Identifying the language of the page or page elements will allow screen readers to read the content in the appropriate language. It also facilitates automatic translation of content.

2. Contrast Errors (Fifty 54 of them) - Very low contrast between text and background colors.

Suggestion: Providing adequate contrast of text for all users, especially users with low vision. The contrast between the foreground (text) color and the background color should be increased. In regards to text size, large text (larger than 18 point or 14 point bold) does not require as much contrast as smaller text.

3. There is justified text in 6 places.

Suggestion: Remove the full justification from the text. This is because large blocks of justified text can negatively impact readability due to varying word/letter spacing and 'rivers of white' that flow through the text.

4. One suspicious alternative text

Suggestion: If the alternative text for an image does not provide the same content or information conveyed by the image, that content will not be available to screen reader users and when images are unavailable. The alternative text for the image or image input should provide an appropriate and equivalent alternative to the content and function of the image. Screen readers and browser presentation inform the user that the object is an image, so alternative text of "image of..." (and similar) should be avoided. If the image does not convey content or if the content is presented in nearby text (e.g., a caption), null/empty alternative text (alt="") will be appropriate.

5. Images having a title attribute value but no alt value.

Suggestion: Ensure the title attribute value presents the content and function of images. For improved accessibility, the alt attribute should be used when possible.

6. The web page contains Links to PDF documents.- PDF documents often have accessibility issues and typically require using a separate application or plug-in to view them. This can cause confusion and navigation difficulties.

Suggestion:

Ensuring the PDF document is natively accessible.

Also informing users that the link will open a PDF document.

-HTML content should often be used in place of or in addition to the PDF document.

7. Alternative text is present for 2 images images that are within link.

Suggestion: including appropriate alternative text on an image within a link to ensure that the function and purpose of the link and the content of the image is available to screen reader users or when images are unavailable.

8. Some headings are not structured well in the page

Suggestion: This can be improved by ensuring that where headings are used, the text in question is actually a heading and that it is structured correctly in the page outline. When this is put into consideration, the headings will facilitate page navigation for users of assistive technologies. The headings will also provide semantic and visual meaning and structure to the document.

9. Un Ordered lists were used

Suggestion: Where list items are sequential or numbered, an ordered list is likely more appropriate.
We should use unordered (bulleted) lists if they are appropriate for the context.

10. Little information is presented on the website footer.

Suggestion: Including more information in the footer will provide site visitors with a sense of consistency, as the same information will appear at the bottom of every single page. This helps people know that they’re really on the web site and makes it easy for repeat visitors to find what they’re looking for. I suggest that some calls to action be added to the footer for example subscribe to our newsletter, learn more, get in touch, leave a review, Share this post and others, links to social accounts among others.
-Considering color contrast while adding information in the footer. Making sure text and images are different enough from your footer background that they can be easily consumed. Avoiding dark text on a dark background or light text on a light background. Using more than just color to differentiate links from other text. (this can be done with underlines, icons, and other cues) Including alt text for any images or graphics in the footer. Screen readers with visual impairments will use these to “read” images
-Also by Sticking to simple, easy-to-read fonts that everyone can consume while making sure they’re large enough so that anyone with vision difficulties can still read every word.

11. An Arial Lable was used.
![wave evaluation of ipython documentation](https://user-images.githubusercontent.com/87822858/198733926-67203c09-4197-4ccf-a6f9-d31b0fdeb8bc.JPG)

Suggestion: I suggest that where possible, the standard HTML label should be used.

### The HTML element to be changed

```html
The Language attribute Missing.

Current Html is 
<html xmlns="http://www.w3.org/1999/xhtml">
```

### Your proposed HTML (or CSS), after change

```html
<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
```

### More info

Add the lang attribute to the starting html tag to declare the language of the page
